docs/패키지_구분_방식.md

```
@@ -0,0 +1,37 @@
+## Domain Package 내부 구조
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)